src/framework/global/dataformatter.cpp

```diff
@@ -44,37 +44,36 @@ String DataFormatter::formatTimeSince(const Date& date)
         return mtrc("global", "Today");
     }
 
-    if (days == 1) {
-        return mtrc("global", "Yesterday");
-    }
-
     if (days < 7) {
+        //: Make sure to translate singular and plural form and whatever else your language has here.
+        //: No need to handle the 0 case here, nor any value > 6,
+        //: for singular best translate to the equivalent of "Yesterday")
         return mtrc("global", "%n day(s) ago", nullptr, days);
```

Comment:
I think it is best to get a translation for all plural forms. Transifex won't let you save without it anyway.

It may be safer to state the singular case as:
> You may translate singular cases to the equivalent of "Yesterday" instead of the "1 day ago" form, but only do this if you are sure that the translation is only used for a single value of "%n".

And also give the link to that Wiki page.

---

FWIW, if we wanted to do this properly we would probably code it like this:
```C++
//: Translate as "[no-equivalent]" if your language doesn't have a single word / phrase
//: that means "yesterday" other than the literal equivalent of "1 day ago".
String yesterday = mtrc("global", "Yesterday");

//: Translate as "[no-equivalent]" if your language doesn't have a single word / phrase
//: that means "2 days ago" in your language other than the literal equivalent of "2 days ago".
String yesteryesterday = mtrc("global", "Yesteryesterday");

//: Translate as "[no-equivalent]" if your language doesn't have a single word / phrase
//: that means "3 days ago" in your language other than the literal equivalent of "3 days ago".
String yesteryesteryesterday = mtrc("global", "Yesteryesteryesterday");

if (days == 1 && yesterday != "[no-equivalent]") {
        return yesterday;
    }

if (days == 2 && yesteryesterday != "[no-equivalent]") {
        return yesteryesterday;
    }

if (days == 3 && yesteryesteryesterday != "[no-equivalent]") {
        return yesteryesteryesterday;
    }

//: Please provide a translation for the singular and all plural forms available in your language.
//: See https://github.com/musescore/MuseScore/wiki/Help-translate-MuseScore#plural-forms
return mtrc("global", "%n day(s) ago", nullptr, days);
```
But that's clearly more trouble than it's worth for something like this.

Reply:
Check again, please